ci/autoresearch/gpio.py

```
@@ -65,24 +65,24 @@ async def run_gpio_pretest(
+                    # to block until the device starts producing serial output
                     if serial_interface is not None:
-                        print("  Resetting device via SerialInterface.reset_device()...")
+                        print("  Resetting device via SerialInterface.reset_device(wait_for_output=True)...")
                         await serial_interface.reset_device(board=None)
                     else:
                         from ci.util.serial_interface import _pyserial_dtr_reset
 
                         _pyserial_dtr_reset(port)
-
-                    await asyncio.sleep(3.0)
+                        await asyncio.sleep(1.0)
```

Ruff (0.15.10)

[error] Undefined name `asyncio`

(F821)